package.json

```diff
@@ -63,11 +64,12 @@
 		"typescript": "^5.2.2",
 		"vite": "^4.5.0",
 		"vitest": "^0.34.6",
-		"vue": "^3.4.38",
+		"vue": "^3.5.13",
 		"vue-tsc": "^2.0.29"
 	},
 	"dependencies": {
 		"@videodb/player-vue": "~0.0.3",
+		"@vue/compiler-sfc": "^3.5.13",
```

Comment:
@tanishq3105 
`@vue/compiler-sfc` isn't required here because it's already included with Vue.js 3 by default.

> Note: as of 3.2.13+, this package is included as a dependency of the main vue package and can be accessed as vue/compiler-sfc. This means you no longer need to explicitly install this package and ensure its version match that of vue's. Just use the main vue/compiler-sfc deep import instead.

Source: https://www.npmjs.com/package/@vue/compiler-sfc